specification/batch/data-plane/Microsoft.Batch/stable/2018-03-01.6.1/BatchService.json

```diff
@@ -11533,7 +11533,7 @@
         "commandLine": {
           "type": "string",
           "title": "The command line of the Job Manager task.",
-          "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
+          "description": "Tasks should be idempotent. For more information, please see TaskContainerSettings.maxTaskRetryCount. The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
```

Comment:
Avoid this style of "reference" since it doesn't look nice in any generated language. I think we need to think of a better way to do this.

This comment applies down the line.